turbo.json

```diff
@@ -11,10 +11,12 @@
       "persistent": true
     },
     "lint": {
-      "dependsOn": ["^build"]
+      "dependsOn": ["^build"],
+      "cache": false
```

Comment:
Is there a reason that we'd want the lint to run fresh every time?

Reply:
I figure linting is a non deterministic task. This top level lint will execute lint in all apps, packages, etc. Succeeding once does not mean you are good to go since one change could cause the linter to break.

That said, unlinted files should not be possible with husky so over time everything funnels itself to cleaned up. Let me know if you want to revert

Reply:
> ...one change could cause the linter to break.

If any file changed/got added/got deleted in a given workspace, shouldn't that bust the cache and thus run clean?

The only edge case I think we'd need to consider is handling an update to the base ESLint configuration [here](https://github.com/colorstackorg/oyster/blob/main/config/eslint/base.js). Since Turbo only uses a workspace's files as the cache input, we'd probably need to add that base config as an input as well since that's outside of all the workspaces. I just peeped in the Turbo [documentation](https://turbo.build/repo/docs/core-concepts/caching/file-inputs#customizing-the-behavior) that they allow you to add on top of their default using the `$TURBO_DEFAULT$` keyword. Thinking we can do something like this?

```json
{
  "lint": {
    "dependsOn": ["^build"],
    "inputs": ["$TURBO_DEFAULT$", "./config/eslint/*"]
  },
  "lint:fix": {
    "dependsOn": ["^build"],
    "inputs": ["$TURBO_DEFAULT$", "./config/eslint/*"]
  }
}
```

Reply:
Ah I see. I like the turbo defaults but agree we should default to Turbo's own configuration since it has likely been more thought through. Reverting.